geomstats/geometry/stiefel.py

```diff
@@ -356,21 +356,7 @@ def log(self, point, base_point, max_iter=30):
             Tangent vector at the base point equal to the Riemannian logarithm
             of point at the base point.
         """
-        point = gs.to_ndarray(point, to_ndim=3)
-        n_points, _, _ = point.shape
-
-        base_point = gs.to_ndarray(base_point, to_ndim=3)
-        n_base_points, _, p = base_point.shape
-
-        if not (n_points == n_base_points
-                or n_points == 1
-                or n_base_points == 1):
-            raise NotImplementedError
-
-        if n_base_points == 1:
-            base_point = gs.tile(base_point, (n_points, 1, 1))
-        if n_points == 1:
-            point = gs.tile(point, (n_base_points, 1, 1))
+        _, _, p = base_point.shape
```

Comment:
what about `p = base_point.shape[-1]` so it works with only 1 2d-matrix?

Reply:
done.